### DISCUSSION TITLE
Is there any way to connect Android app to wallet apps?

### DISCUSSION THREAD

**Discussion by quanqx** [bloxbean/cardano-client-lib#387] [Q&A]
Hi there,
I'm trying to find a wallet that supports Cardano and provides API to connect Android dApp.
If you know, could you please share me?
Thank you so much..!

**Comment by satran004** [comment]
Hi @quanqx ,
Did you try Eternl or Flint wallet ? Though I am not sure how the integration works on Android.

**Reply by quanqx** [reply]
Hi @satran004 ,
As my research, both above wallet have no APIs to connect.
I will continue to check others.
Thank you for you response!
